### PR TITLE
Adding new content and some editing

### PR DESCRIPTION
**Pull request by taleldayekh**

### CODE REVIEW

File: docs/sdk-docs/installing-dapp/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+```bash
+git clone --recurse-submodules https://github.com/raiden-network/light-client.git
+```
+
+## Build the SDK
+
+Change to the `raiden-ts` directory, install the dependencies and build the SDK.
+
+```bash
+cd ../raiden-ts
```
Comment (nephix) [inline docs/sdk-docs/installing-dapp/README.md]: Not `cd light-client/raiden-ts`?

File: docs/sdk-docs/installing-dapp/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+npm install
+npm run build
```
Comment (nephix) [inline docs/sdk-docs/installing-dapp/README.md]: I never ran `npm run build`, @kelsos is this necessary?

File: docs/sdk-docs/installing-dapp/README.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+```
+
+## Install the dApp dependencies
+
+Change to the `raiden-dapp` directory and install its dependencies.
+
+```bash
+cd ../raiden-dapp
+npm install --save raiden-ts
```
Comment (nephix) [inline docs/sdk-docs/installing-dapp/README.md]: Isn't `npm install` enough? `--save` adds it to `package.json` where it already is

File: docs/sdk-docs/installing-sdk/README.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+```bash
+npm install raiden-ts
+```
+
+After installing the SDK you can simply import Raiden to your JavaScript or TypeScript project.
+
+```typescript
+import { Raiden } from 'raiden-ts';
+
+# async factory
```
Comment (nephix) [inline docs/sdk-docs/installing-sdk/README.md]: `#` is not valid JavaScript

File: docs/sdk-docs/prerequisites/README.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# Prerequisites
+
+#### For building the SDK and installing the dApp you'll need:
+- Node.js 10+ installed on your system.
+- A Web3 enabled browser, i.e. Chrome with the MetaMask extension.
```
Comment (nephix) [inline docs/sdk-docs/prerequisites/README.md]: Maybe link to MetaMask for convenience

File: docs/sdk-docs/installing-dapp/README.md (hunk 1 above)
Comment (kelsos) [reply docs/sdk-docs/installing-dapp/README.md]: This assumes that you are already in the directory where you cloned the project. Maybe add the information about moving to the directory where the project was cloned after the `git clone` command

File: docs/sdk-docs/installing-dapp/README.md (hunk 3 above)
Comment (taleldayekh) [reply docs/sdk-docs/installing-dapp/README.md]: Yes, your right I've always used only npm install and it works fine.

File: docs/sdk-docs/installing-dapp/README.md (hunk 2 above)
Comment (kelsos) [reply docs/sdk-docs/installing-dapp/README.md]: It was at some point, I still run it but maybe just to be sure @taleldayekh can create a new environment and go through the steps one by one to make sure that they are needed and up to date.

Comment (taleldayekh) [reply docs/sdk-docs/installing-dapp/README.md]: I can confirm that it works without `npm run build` :)

File: docs/sdk-docs/installing-dapp/README.md (hunk 3 above)
Comment (kelsos) [reply docs/sdk-docs/installing-dapp/README.md]: This was required at least once in the past due to npm link shenanigans. I am not sure if it is need anymore or if it is required only the first time you clone the project. Or the first time you start in a new environment. In my experience so far, if you set up the link once then it pretty much works normally. But if you try a new system that never had the project issues might appear.

A good way to check would be in a VM that never worked with the light client to ensure that there is no leftover setup. 

These instructions target someone that never worked with the SDK before so we have to make sure that they work in such an environment.

File: docs/sdk-docs/installing-dapp/README.md (hunk 2 above)
Comment (nephix) [reply docs/sdk-docs/installing-dapp/README.md]: I usually just do `npm install` and I'm all set for local development. I think™ `npm run build` is necessary for publishing to npm

File: docs/sdk-docs/installing-sdk/README.md (hunk 4 above)
Comment (kelsos) [reply docs/sdk-docs/installing-sdk/README.md]: Yes, we have a few parts with python comments also in the rest of the docs. It' is not only here.
All these code snippets should be updated. We should create an issue about that.

File: docs/sdk-docs/installing-dapp/README.md (hunk 1 above)
Comment (nephix) [reply docs/sdk-docs/installing-dapp/README.md]: I would make it 5yo-proof and write down the exact steps :)

Besides that even if you are already in the cloned project, `cd ../raiden-ts` would not work

File: docs/sdk-docs/installing-dapp/README.md (hunk 2 above)
Comment (kelsos) [reply docs/sdk-docs/installing-dapp/README.md]: I would still prefer it if we can verify in a clean VM that this actually works as intended when there is no previous state left. 

I had a lot of cases where something worked on my development environment properly but started failing when someone new tried to run the dApp locally in a new environment.

File: docs/sdk-docs/installing-dapp/README.md (hunk 3 above)
Comment (taleldayekh) [reply docs/sdk-docs/installing-dapp/README.md]: I will leave this one then. I usually clone the repo and install the all dependencies  from scratch each time I start something new on the LC, dunno if that is the same as installing it on another machine or VM.

Comment (nephix) [reply docs/sdk-docs/installing-dapp/README.md]: I never ran it that way, but yes we should create a ticket and go through the steps on a clean machine and verify that they work (or not)

**Comment by kelsos** [comment]
@taleldayekh Any change you can open an issue about verifying that the guide works as expected in a fresh environment? If you need any help with that I am able to provide assistance. 

You can try all the steps that @nephix mentioned and assuming that they work properly you can also adjust the guide accordingly. The most probable scenario is that our guide is a bit outdated but I am a little cautious since I have been bitten by such changes in the past where my existing environment skewed the results.

**Comment by nephix** [comment]
I have created an issue #845

**Pull request merged**